.github/workflows/company4-e2e.yml

```diff
@@ -0,0 +1,105 @@
+name: E2E Testing
+
+on:
```

Comment:
Add a `pull_request` under here so we can see this running within the PR perhaps?

Reply:
Done